py_mmd_tools/adc_elements.yaml

```diff
@@ -0,0 +1,396 @@
+ADC_global_attributes:
```

Comment:
is this line needed? It seems to be only this element at the first level? Maybe call it ACDD_global_attributes if kept?